### PR TITLE
Added new descriptions and updated images

### PR DESCRIPTION
**Pull request by dsimon1992**
@gsanchezu to review the changes as well.

### CODE REVIEW

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md [hunk 1]
```diff
@@ -68,6 +69,10 @@ All Cost Centers and Workspaces belong to an Organization.
 
 The administrators of an Organization can change many settings that affect the whole organizations. See more details [here](../Administering Your Organization/admin-overview.md).
 
+### Favorites
+
+The Favorites section contains a list of any workspace, cost center or organization that the user added for easier or more convenient access to. To add to this section, simply click the Star icon in the upper left hand corner of any of the aforementioned spaces. To remove, click the Star icon again.
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]: Hey @dsimon1992, what about mentioning that the star will appear highlighted in the items you have added to Favorites?

File: Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md [hunk 2]
```diff
@@ -94,12 +96,32 @@ ___
...
-* Connect to a provider to orchestrate deployments. Click **New Provider** to add AWS, Azure, vSphere, Google Cloud, OpenStack, or CloudStack.
+* Connect to a provider to orchestrate deployments. Click **New** to add AWS, Azure, vSphere, Google Cloud, OpenStack, or CloudStack.
 * Locate a provider through search or by type.
 * Sync or delete provider accounts using Bulk Actions.
 
 ___
 
+**Catalog**
+
+![getting-started-login-9.png](../../images/cloud-application-manager/getting-started-login-9.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md]: Hey @dsimon1992, please use a more complete screenshot of the Catalog. I guess you took it from a local env, but please use production to take this one

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md (hunk 1 above)
Comment (dsimon1992) [reply Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]: Updated the description. Let me know if its suitable.

File: Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md (hunk 2 above)
Comment (dsimon1992) [reply Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md]: Updated the screenshot.

**Comment by lyzalou** [comment]
@asanokhin this is ready to be merged.

**Pull request merged**